docs/FAQ.rst

```diff
@@ -249,10 +265,11 @@ Avalon uses sealed data internally for storing private enclave signing and
 encryption keys.
 As result application specific data size is not dependent on the sealed data
 storage.
-It is indirectly limited by the maximum enclave size
+It is indirectly limited by the maximum Enclave Page Cache (EPC) size
 (enclave includes both data and code).
-The maximum enclave size is limited to 128Mbytes.
-Enclaves can be bigger but it results in swapping in and out of the enclave,
+The maximum EPC size is limited to 128 Mbytes on Intel Xeon E3 and
+256 Mbytes on Intel Xeon E Mehlow-R.
```

Comment:
Should we mention Intel Xeon E-2200 instead of refer Intel Xeon E Mehlow-R ?

Reply:
Yes.  I made the change in for the next PR I will make for the FAQ.